apps/golden-sample-app-e2e/test-config.ts

```diff
@@ -0,0 +1,23 @@
+import { User } from './data/data-types/user';
+import config from '../../playwright.config';
+import 'dotenv/config';
+
+export class TestConfig {
+  baseUrl: string = config.use?.baseURL || 'BASE URL UNDEFINED';
+  locale: string = 'en';
```

Comment:
```suggestion
  locale = 'en';
```

Reply:
If we assign value to a variable then typescript infers the type assigned to `locale`